projects/core/src/modules/numeric/numeric.pipe.ts

```diff
@@ -67,7 +90,7 @@ export class SkyNumericPipe implements PipeTransform, OnDestroy {
       config.truncate === false &&
       config.digits === undefined
     ) {
-      config.digits = 0;
+      options.digits = 0;
```

Comment:
Changed this in the latest update (and the one a few lines below) as we follow this up by setting the options to this value and this was causing caching side effects if the consumer was using the same object in memory for future calls.